### PR TITLE
Add Dockerfile for building test environment

### PR DESCRIPTION
**Pull request by leonwanghui**
Signed-off-by: leonwanghui <wanghui71leon@gmail.com>

## Introduction
This patch mainly focuses on solving the issue #31 

## How to build mock-broker and test-job docker image
These commands below present how to building test environment:
```shell
make docker
```

After this finished, you will find two docker images created:
```shell
root@vultr:~/workspace/osb-checker# docker images
REPOSITORY                         TAG                 IMAGE ID            CREATED             SIZE
osb-checker/test-job               2.13                bd833779fd29        6 seconds ago       676MB
osb-checker/mock-broker            2.13                deb24670b84c        10 seconds ago      672MB
```

## How to run these two docker images
```shell
docker run -d --net=host osb-checker/mock-broker:2.13
docker run -it --net=host osb-checker/test-job:2.13
```

### CODE REVIEW

File: 2.13/tests/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  sudo \
+  wget \
+  make \
+  g++ \
+  && rm -rf /var/lib/apt/lists/* \
+  && apt-get clean
+RUN wget --no-check-certificate https://deb.nodesource.com/setup_8.x \
+  && chmod +x setup_8.x && ./setup_8.x \
+  && apt-get install -y nodejs
```
Comment (n3wscott) [inline 2.13/tests/Dockerfile]: I am sure there is a blessed node docker image.

**Comment by norshtein** [comment]
The PR mostly LGTM. One thing is, better to create a doc and add the description about how to use the docker image to the doc.

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,5 @@
+VERSION_NUMBER := 2.13
```
[the remark this answers is not in the document]
Comment (leonwanghui) [reply Makefile]: I think for now it can be set to `2.13` statically.

[the remark this answers is not in the document]
Comment (zhongyi-zhang) [reply Makefile]: That's OK.

**Comment by norshtein** [comment]
LGTM

**Pull request merged**